src/module.rs

```diff
@@ -623,7 +623,6 @@ impl ModuleInstance {
 			}
 		};
 
-		check_function_args(func_instance.signature(), &args)?;
```

Comment:
In this case, there is not a lot of sense in actually formatting error message inside of `check_function_args`. Maybe we can get rid of it?